### PR TITLE
fix(test): initial POC for BDD booster service integration tests

### PR DESCRIPTION
**Pull request by ldimaggi**

### CODE REVIEW

File: booster_api/features/src/support/helpers.py [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+import json, jmespath, operator
+import constants
+
+count = 1
+
+def create_space_name(template="SDD"):
+    var = datetime.datetime.now()
+    var = var.isoformat().rsplit('.')[0]
+    space = constants.launch_detail.userid_primary + "-" + template + "-space-" + var
+    space = space.replace('@','-')
```
Comment (rgarg1) [inline booster_api/features/src/support/helpers.py]: I like this change. Thanks for bringing this in, Len. I ignored that someone's user ID could also be an email address.

File: booster_api/features/src/support/planner_jsons/add_wi_comment.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"data":{"type":"comments","attributes":{"body":"$comment_text","markup":"Markdown"},"links":{}}}
```
Comment (rgarg1) [inline booster_api/features/src/support/planner_jsons/add_wi_comment.json]: You probably don't need the directory _planner_jsons_

Comment (rgarg1) [reply booster_api/features/src/support/planner_jsons/add_wi_comment.json]: Create a new directory _booster_jsons_ and use that instead.

File: booster_api/features/src/support/planner_jsons/new_import.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{"data":{"gitRepository":"test123","projectName":"test123","pipeline":"maven-release","space":"adb3437-d4f4-482e-8459-02ccb0561ea7"}}
```
Comment (rgarg1) [inline booster_api/features/src/support/planner_jsons/new_import.json]: Avoid hardcoding dynamic data like proj name, git repo name, space ID etc. 
You can use a placeholder in your json and then use the replace function to overwrite it (in memory) with real dynamic data.

File: booster_api/features/src/support/helpers.py (hunk 1 above)
Comment (ldimaggi) [reply booster_api/features/src/support/helpers.py]: Thanks!

File: booster_api/features/src/support/planner_jsons/add_wi_comment.json (hunk 2 above)
Comment (ldimaggi) [reply booster_api/features/src/support/planner_jsons/add_wi_comment.json]: Good idea - done!

**Pull request merged**